cmp/tab/TabContainerModel.js

```diff
@@ -88,6 +88,8 @@ export class TabContainerModel {
         
         this.selectedId = child ? id : children[0].id;
 
+        if (child.reloadOnShow) this.setLastRefreshRequest(Date.now());
```

Comment:
Comment was covered up by rename, but still relevant:

lbwexler 25 minutes ago  Owner
This seems wrong to me. If any single child pane with this option is selected -- all panes in the container that are subsequently shown will get an extra refresh as well.

  @febbraiod
febbraiod 17 minutes ago  Member
Yeah I noticed that too. I suppose we could call the child component's loadAsync directly, but that seemed wrong too as the tab pane's doc strings indicate:

Reloads the active tab whenever its lastLoaded is out of date with its parent's 'lastRefreshRequest'.
and
Contained components that load data/state from the server should implement loadAsync(), but
generally leave the calling of that method to this component